jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -380,6 +380,10 @@ ecma_string_t *ecma_string_substr (const ecma_string_t *string_p, ecma_length_t
 void ecma_string_trim_helper (const lit_utf8_byte_t **utf8_str_p,
                               lit_utf8_size_t *utf8_str_size);
 ecma_string_t *ecma_string_trim (const ecma_string_t *string_p);
+ecma_value_t ecma_string_pad (ecma_value_t original_string_p,
```

Comment:
If the implementation is guarded then the function prototype must be also guarded. (I think the Doxygen error is due to this).